### PR TITLE
add vote.alphadao.money

### PR DESCRIPTION
**Pull request by 0xmatt1**
#### Hi! What is your PR about?

- [ ] Add or edit a skin
- [x] Add a custom domain for your space
- [ ] Add an alias to migrate your space

### CODE REVIEW

File: spaces/domains.json [hunk 1]
```diff
@@ -478,4 +478,5 @@
   "vote.aavegotchi.com": "aavegotchi.eth",
   "snapshot.keplerdao.finance": "keplerdao.eth",
   "signal.paladin.vote": "palvote.eth"
+  "vote.alphadao.money": "alphagov.eth"
```
Comment (gokaiorg) [inline spaces/domains.json:481]: A comma is missing at the end of the line above your sub-domain.
```suggestion
  "signal.paladin.vote": "palvote.eth",
  "vote.alphadao.money": "alphagov.eth"
```

**Review by gokaiorg — CHANGES_REQUESTED** [review]

**Comment by 0xmatt1** [comment]
Updated requested changes

**Comment by gokaiorg** [comment]
There is a conflict in your PR since the `domains.json` file has been updated, can you resolve it?
To prevent conflicts, it is recommended to add your subdomain between two existing domains rather than at the end or beginning of the list.

**Comment by 0xmatt1** [comment]
updated, moved our entry a few lines up

**Review by gokaiorg — APPROVED** [review]

**Pull request merged**